.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
```

Comment:
I'm not sure whether this should have a more specific name than "CI".  One one hand, the only check is currently for pr-blocking-todos, and it would be nice to list that specifically in the machinery of failed checks.  On the other hand, if the github CI stage is expanded in the future, this less specific name would be appropriate.

Also, I'm not 100% sure github allows the name to be different, but it's worth a try.

Reply:
I think changing the name, at least for now, is a good idea.  If we ever add more CI through github actions in the same script, we can reevaluate the name.